doc/get_started.rst

```diff
@@ -3,110 +3,89 @@
-benchmarks can be retrieved from GitHub or created locally. To discover which
-benchmarks are presently available look for
-`benchmark_* repositories on GitHub <https://github.com/benchopt/>`_,
-such as for `Lasso -- l1-regularized linear regression <https://github.com/benchopt/benchmark_lasso>`_.
-This benchmark can be retrieved locally with:
+    pip install -U -i https://test.pypi.org/simple/benchopt
 
-.. prompt:: bash $
+.. attention::
 
-    git clone https://github.com/benchopt/benchmark_lasso.git
+   The **development version** is a work in progress and hence might contain incomplete features.
+   A typical user is advised to use the **stable version** instead.
 
+With benchopt being installed, you get access to the :ref:`Command Line Interface (CLI) <cli_ref>`,
+which enables simple and easy manipulation of benchmarks just from the terminal.
 
 
-Run existing benchmark
-----------------------
+Run an existing benchmark
+-------------------------
 
-This section illustrates benchopt's command line interface on the `Lasso benchmark <https://github.com/benchopt/benchmark_lasso>`_; the syntax is applicable to any benchmark.
-All this section assumes that you are in the parent folder of the ``benchmark_lasso`` folder.
-The ``--env`` flag specifies that everything is run in the ``benchopt_benchmark_lasso`` ``conda`` environment.
-
-**Installing benchmark dependencies**: benchopt exposes a CLI to install solvers' dependencies automatically.
-It only works inside a ``conda`` environment. To install all requirements of the benchmark, make sure a ``conda``
-environment is activated and run:
-
-.. prompt:: bash $
+Let's get the first steps with benchopt by comparing some solvers of
+`Lasso problem <https://en.wikipedia.org/wiki/Lasso_(statistics)>`_ on a
+`Leukemia dataset <https://www.science.org/doi/10.1126/science.286.5439.531>`_.
 
-    benchopt install --env ./benchmark_lasso
+Benchopt community maintains :ref:`several optimization benchmarks <available_benchmarks>`
+and thrives at making them accessible and up to date, so as for the Lasso problem.
 
-**Run a benchmark**: to run benchmarks on all datasets and with all solvers, run:
+Start by cloning the Lasso benchmark repository and then ``cd`` to it.
 
 .. prompt:: bash $
 
-    benchopt run --env ./benchmark_lasso
-
-The command ``benchopt run`` can also be used outside of a ``conda`` environment without the flag ``-e/--env``.
-In that case, the benchmark will only run solvers that are currently installed.
+    git clone https://github.com/benchopt/benchmark_lasso.git
+    cd benchmark_lasso
 
-**Run only some solvers and datasets**: to run only the ``sklearn`` and ``celer`` solvers, on the ``simulated`` and ``finance`` datasets, run:
+Then install automatically the benchmark requirements, namely, the solvers `skglm <https://contrib.scikit-learn.org/skglm/>`_ and
+`scikit-learn <https://scikit-learn.org/stable/>`_, and then the dataset Leukemia. 
```

Comment:
why not do it in one call?

Reply:
Yess, indeed! That would be much simplier